packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.css

```diff
@@ -1,3 +1,16 @@
+/* 
+ * Styling hooks are not supported by the LWC Local Development Server.
+ * You can safely comment those out in order to run the code in the Local Dev Server, these are for a11y purpose only.
+ * Issue opened to Salesforce: https://github.com/forcedotcom/salesforcedx-vscode/issues/4493
+ */
+:host {
```

Comment:
Given these are in the exampleSearch, a user would need to chose to include them in their page. Maybe we should add a comment saying this is to increase contract for accessibility purposes?

Reply:
Added a comment, great idea!